### ISSUE TITLE
Unable to access app permissions

### ISSUE THREAD

**Issue by Kate-T** [anbox/anbox#851]
1. Please check that no similar bug is already reported. Have a look on the list of open bugs at https://github.com/anbox/anbox/issues
2. Make sure you are running the latest version of Anbox before reporting an issue. Update snap to latest: `snap refresh --devmode --edge anbox`
3. Make sure you have debug logs enabled:
`sudo snap set anbox debug.enable=true`
4. Reproduce the error while debug logs enabled.
5. Run the anbox logs collection utility and attach the tar file.
`sudo /snap/bin/anbox.collect-bug-info `

/home/kathleen/anbox-system-diagnostics-2018-08-07.zip 
(BTW, Why is this a zip file instead of a tar file?)

6. ** Please paste the result of `anbox system-info` below:**
```
[version: 4
snap-revision: 134
cpu:
  arch:  x86
  brand: Intel(R) Core(TM) i5-4210U CPU @ 1.70GHz
  features: 
    - aes
    - sse4_1
    - sse4_2
    - avx
    - avx2
os:
  name: Ubuntu
  version: 16.04.5 LTS (Xenial Xerus)
  snap-based: true
kernel:
  version: Linux version 4.4.0-131-generic (buildd@lgw01-amd64-015) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.10) ) #157-Ubuntu SMP Thu Jul 12 15:51:36 UTC 2018
  binder: true
  ashmem: true
graphics:
  egl:
    vendor: Mesa Project
    version: 1.4 (DRI2)
    extensions:
      - EGL_CHROMIUM_sync_control
      - EGL_EXT_create_context_robustness
      - EGL_EXT_image_dma_buf_import
      - EGL_EXT_image_dma_buf_import_modifiers
      - EGL_KHR_config_attribs
      - EGL_KHR_create_context
      - EGL_KHR_create_context_no_error
      - EGL_KHR_fence_sync
      - EGL_KHR_get_all_proc_addresses
      - EGL_KHR_gl_colorspace
      - EGL_KHR_gl_renderbuffer_image
      - EGL_KHR_gl_texture_2D_image
      - EGL_KHR_gl_texture_3D_image
      - EGL_KHR_gl_texture_cubemap_image
      - EGL_KHR_image
      - EGL_KHR_image_base
      - EGL_KHR_image_pixmap
      - EGL_KHR_no_config_context
      - EGL_KHR_reusable_sync
      - EGL_KHR_surfaceless_context
      - EGL_EXT_pixel_format_float
      - EGL_KHR_wait_sync
      - EGL_MESA_configless_context
      - EGL_MESA_drm_image
      - EGL_MESA_image_dma_buf_export
      - EGL_NOK_swap_region
      - EGL_NOK_texture_from_pixmap
      - EGL_NV_post_sub_buffer
      - EGL_WL_bind_wayland_display
  gles2:
    vendor: Intel Open Source Technology Center
    vendor: OpenGL ES 3.0 Mesa 18.0.5
    extensions:
      - GL_ANGLE_texture_compression_dxt3
      - GL_ANGLE_texture_compression_dxt5
      - GL_APPLE_texture_max_level
      - GL_EXT_blend_func_extended
      - GL_EXT_blend_minmax
      - GL_EXT_clip_cull_distance
      - GL_EXT_color_buffer_float
      - GL_EXT_compressed_ETC1_RGB8_sub_texture
      - GL_EXT_discard_framebuffer
      - GL_EXT_disjoint_timer_query
      - GL_EXT_draw_buffers
      - GL_EXT_draw_buffers_indexed
      - GL_EXT_draw_elements_base_vertex
      - GL_EXT_frag_depth
      - GL_EXT_map_buffer_range
      - GL_EXT_multi_draw_arrays
      - GL_EXT_occlusion_query_boolean
      - GL_EXT_polygon_offset_clamp
      - GL_EXT_read_format_bgra
      - GL_EXT_robustness
      - GL_EXT_separate_shader_objects
      - GL_EXT_shader_integer_mix
      - GL_EXT_texture_border_clamp
      - GL_EXT_texture_compression_dxt1
      - GL_EXT_texture_filter_anisotropic
      - GL_EXT_texture_format_BGRA8888
      - GL_EXT_texture_rg
      - GL_EXT_texture_sRGB_decode
      - GL_EXT_texture_type_2_10_10_10_REV
      - GL_EXT_unpack_subimage
      - GL_INTEL_performance_query
      - GL_KHR_blend_equation_advanced
      - GL_KHR_context_flush_control
      - GL_KHR_debug
      - GL_KHR_no_error
      - GL_KHR_robust_buffer_access_behavior
      - GL_KHR_robustness
      - GL_MESA_shader_integer_functions
      - GL_NV_draw_buffers
      - GL_NV_fbo_color_attachments
      - GL_NV_read_buffer
      - GL_NV_read_depth
      - GL_NV_read_depth_stencil
      - GL_NV_read_stencil
      - GL_OES_EGL_image
      - GL_OES_EGL_image_external
      - GL_OES_EGL_sync
      - GL_OES_compressed_ETC1_RGB8_texture
      - GL_OES_depth24
      - GL_OES_depth_texture
      - GL_OES_depth_texture_cube_map
      - GL_OES_draw_buffers_indexed
      - GL_OES_draw_elements_base_vertex
      - GL_OES_element_index_uint
      - GL_OES_fbo_render_mipmap
      - GL_OES_get_program_binary
      - GL_OES_mapbuffer
      - GL_OES_packed_depth_stencil
      - GL_OES_required_internalformat
      - GL_OES_rgb8_rgba8
      - GL_OES_sample_shading
      - GL_OES_sample_variables
      - GL_OES_shader_multisample_interpolation
      - GL_OES_standard_derivatives
      - GL_OES_stencil8
      - GL_OES_surfaceless_context
      - GL_OES_texture_3D
      - GL_OES_texture_border_clamp
      - GL_OES_texture_float
      - GL_OES_texture_float_linear
      - GL_OES_texture_half_float
      - GL_OES_texture_half_float_linear
      - GL_OES_texture_npot
      - GL_OES_texture_stencil8
      - GL_OES_vertex_array_object
      - GL_OES_vertex_half_float
]
```

**Please describe your problem:**
When I go into Settings and then Apps, I choose an app so I can find the permissions to that app. Instead of finding permissions, when I click on the app from the list, I am immediately kicked back out to the general settings page. I'm unable to access app permissions at all.

**What were you expecting?:**
I expected to be able to access the choices of permissions for an app (Google Play Services and Google Play Store to enable all permissions for those two apps.

**Additional info:**
Now that I have changed from the beta version of anbox to the edge version (thanks, Morphis!), I no longer have Google Play Services in my anbox list of apps. It's gone. What do I have to do to get it back? Reinstall the GApps?

**Comment by morphis** [comment]
> (BTW, Why is this a zip file instead of a tar file?)

Because Github doesn't allow the upload of tar files. You forgot to attach the file here and instead just shared a string show the path where the file is stored on your PC.

> Now that I have changed from the beta version of anbox to the edge version (thanks, Morphis!), I no longer have Google Play Services in my anbox list of apps. It's gone. What do I have to do to get it back? Reinstall the GApps?

I am not sure how you installed GApps but if you modified the android.img the snap comes with it gets overriden everytime the snap is refreshed. Modifying the android.img is not supported. If you want to add changes to the Android root filesystem and want them to persist an update of the snap you have to use https://docs.anbox.io/userguide/advanced/rootfs_overlay.html

> When I go into Settings and then Apps, I choose an app so I can find the permissions to that app. Instead of finding permissions, when I click on the app from the list, I am immediately kicked back out to the general settings page. I'm unable to access app permissions at all.

This sounds like a bug. We need to output of `anbox.collect-bug-info` to say more what the reason for this can be.

**Comment by Kate-T** [comment]
I notice the "needs-info" tag is still on this question. Now that I've submitted the anbox-system-diagnostics, is there some other information I need to provide?

Perhaps I should mention that I have a Lenovo G50-70 laptop, which is running Ubuntu 16.04 LTS.

I should also mention that the problem of Google Play Services being missing has been resolved. I have no idea HOW it was resolved, but as mysteriously as it disappeared it also just now reappeared.
